metricflow/sql/render/expr_renderer.py

```diff
@@ -124,6 +126,21 @@ def visit_function_expr(self, node: SqlAggregateFunctionExpression) -> SqlExpres
             execution_parameters=combined_params,
         )
 
+    def visit_percentile_expr(self, node: SqlPercentileExpression) -> SqlExpressionRenderResult:
```

Comment:
Should we make this as the default implementation so it won't be duplicated among several sql engines?

Reply:
yes - this is currently in the DefaultSqlExpressionRenderer, so it would be the default for snowflake, duckdb, postgres, and databricks.

Reply:
Oops, I was reading the code wrong 😂